CONTRIBUTING.md

```diff
@@ -83,6 +83,13 @@ Before a pull request can be merged:
 * The CI has to pass successfully
 * Every comment has to be addressed and resolved
 
+### Image References
+
+When specifying the image to use for a task step:
+
+* If the release-service-utils image is used, the url should be `quay.io/redhat-appstudio/release-service-utils:$tag` where `$tag` is the Git SHA based tag
+* A Git SHA based tag should be used if available. The `release-service-utils` image will always have this tag available
```

Comment:
I thought this line read weird after the line above was updated. Because I assumed this is also just talking about the utils image. But now I realize this speaks about all images in general, right? But I don't think we agreed on this - we were only talking about the utils image. And tbh I don't even think other images often have such a tag. 

Instead, we should probably say that other images should always use a fixed tag, e.g. registry.access.redhat.com/ubi8/ubi@sha256:c94bc309b197f9fc465052123ead92bf50799ba72055bd040477ded452d87a0d or registry.access.redhat.com/ubi8/ubi:8.8-1067.1698056881, but not registry.access.redhat.com/ubi8/ubi:8.8 or registry.access.redhat.com/ubi8/ubi:latest or registry.access.redhat.com/ubi8/ubi

So here's what I'd propose:
```
Most tasks in this repo use the release-service-utils image defined in [the release-service-utils repo](https://github.com/redhat-appstudio/release-service-utils). When referencing this image, the url should be quay.io/redhat-appstudio/release-service-utils:$tag where $tag is the Git SHA based tag.

For other images, the reference should always either specify an image by its digest (e.g. registry.access.redhat.com/ubi8/ubi@sha256:c94bc309b197f9fc465052123ead92bf50799ba72055bd040477ded) or a non-moving tag should be used (e.g. registry.access.redhat.com/ubi8/ubi:8.8-1067.1698056881). Floating tags like latest or 8.8 in the case of the ubi image should be avoided.
```

WDYT?

Reply:
I changed it slightly but updated to be close to what you suggested

Reply:
Cool, thanks, it looks great!